en/docs/administer/monitoring-the-identity-server.md

```diff
@@ -17,12 +17,11 @@ topics:
 -   SOAP Tracer
 -   Workflow Requests
 
-!!! info "Recommended reading"
-    -   For a detailed list of monitoring options, see the [Monitoring
-        section in the WSO2 Administration
-        Guide](../../administer/capturing-system-data-in-error-situations).
-    -   For information on the Carbon Health Check API to monitor server
-        health, see [Monitoring Server Health in the WSO2 Administration
-        Guide](../../administer/monitoring-server-health).
-    -   For information on monitoring workflows, see [Monitoring Workflow
+!!! info "Recommended reading" - For a detailed list of monitoring
+options, see the [Monitoring         section in the WSO2 Administration
```

Comment:
There should be a line break after !!! info "Recommended reading"

Otherwise, the macro will not appear properly